packages/storybook/.storybook/TOC.tsx

```diff
@@ -0,0 +1,55 @@
+import { mergeStyles } from '@fluentui/react';
+import React, { useState } from 'react';
+import { BackToTop, TableOfContents } from 'storybook-docs-toc';
+
+export const TOC = (props: {children: React.ReactNode}): JSX.Element => {
```

Comment:
can we add a comment here explaining the meaning of TOC?

Reply:
Sure